CMakeLists.txt

```diff
@@ -58,6 +58,8 @@ option(USE_CUDA  "Build with CUDA" OFF)
 option(USE_CUDNN "Build with CUDNN" OFF)
 option(USE_TENSORRT "Build with Tensor RT" OFF)
 option(ENABLE_DATATRANSFORM "Enable datatransform for Sagemaker-scikit-learn-extension models" OFF)
+option(USE_OPENMP "Build with OpenMP" OFF)
```

Comment:
It looks like `option` cannot take strings other than `ON` and `OFF`, should we use `set` here like https://github.com/neo-ai/tvm/blob/dev/cmake/config.cmake#L151.

Reply:
I was able to use `cmake .. -DUSE_MLAS=ON -DUSE_OPENMP=intel`

Reply:
I see, that's overriding the `option` with `set` through command line.

Reply:
I see, I think this is ok since it is consisent with other DLR flags such as USE_OPENCL, USE_TENSORRT which are defined with `option` but can be overriding with `set` from command line.